providers/docker/src/main/scala/restui/providers/docker/DockerClient.scala

```diff
@@ -68,7 +68,7 @@ class DockerClient(private val client: JDockerClient, private val settings: Sett
           findEndpoint(labels, networks).map {
             case (serviceName, address) =>
               if (event.getStatus == StartFilter)
-                ServiceEvent.ServiceUp(Service(serviceName, OpenApiFile(ContentType.fromString(address), address)))
+                ServiceEvent.ServiceUp(Service(serviceName, serviceName, OpenApiFile(ContentType.fromString(address), address)))
```

Comment:
you pass serviceName twice for id and servicename is that what you want?

Reply:
I moved to the container id, I did the same for the Kubernetes provider